12_glob_lab/globlab.txt

```diff
@@ -5,25 +5,44 @@ https://www.tocode.co.il/bundles/unix1/lessons/globlab
 
   1. All files whose name starts with a lowercase letter
   Answer: 
+l */[a-z]*
+
 
   2. All files whose name starts with an uppercase letter
   Answer:
+l */[A-Z]*
+
 
   3. All files whose name ends in an "a", "e", "i", "o" or "u"
   Answer:
 
+l */*[aeiou]
+
   4. All files whose name DOES NOT end in an "a", "e", "i", "o" or "u"
   Answer:
+l */*[!aeiou]
+
 
   5. All files with extension ".c" that are in a folder named "src"
   Answer:
 
   6. All files with extension ".c" under any folder
   Answer:
+echo */*[c]
```

Comment:
No need for square brackets there, but a `.` is required:

1. You don't need square brackets around a single character, as they denote a selection of one character from a list of characters. When you only have a single character you can just use it.

2. The `.` before c is important as not to capture files that just end in c for example `tac`

So the result should be: `echo */*.c`